Std/Lean/Meta/Basic.lean

```diff
@@ -436,3 +436,10 @@ where
       match ← tac goal with
       | none => acc.modify λ s => s.push goal
```

Comment:
```suggestion
      | none => acc.modify fun s => s.push goal
```

Reply:
No, we have mapsto, but never did lambda.

Reply:
Do you really just want to forbid the character `λ` entirely? That seems overly restrictive.

Reply:
Also, the `λ` you've noticed here isn't even part of this PR. Perhaps do it separately?

Reply:
> Do you really just want to forbid the character `λ` entirely? That seems overly restrictive.

It's a style rule within std. It doesn't seem particularly restrictive since `fun` is entirely interchangeable with `λ`, although a linter that just bans the character might have false positives in documentation.